tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile.py

```diff
@@ -201,23 +194,19 @@ def test_rsc_dep_for_scala_java_and_test_targets(self):
       dependee_graph = self.construct_dependee_graph_str(jobs, task)
 
       self.assertEqual(dedent("""
-                     zinc(java/classpath:java_lib) -> {}
+                     zinc[zinc-java](java/classpath:java_lib) -> {}
                      rsc(scala/classpath:scala_lib) -> {
-                       zinc_against_rsc(scala/classpath:scala_lib)
-                     }
-                     zinc_against_rsc(scala/classpath:scala_lib) -> {
-                       zinc(scala/classpath:scala_test)
+                       zinc[zinc-only](scala/classpath:scala_test)
                      }
+                     zinc[rsc-and-zinc](scala/classpath:scala_lib) -> {}
                      rsc(scala/classpath:scala_dep) -> {
```

Comment:
Does this block:
```
rsc(scala/classpath:scala_dep) -> {
  rsc(scala/classpath:scala_lib),
  zinc[rsc-and-zinc](scala/classpath:scala_lib)
}
```
...mean that using `rsc` for a target depends on using both `rsc` and `zinc` for a scala-only dep, or am I misreading? That would be surprising.

Reply:
Replied out of line: https://github.com/pantsbuild/pants/pull/7742#issuecomment-495816508 -- the arrow might be considered to be pointing the wrong direction.

Reply:
Heh, yes: definitely. Thanks!